Documentation/ApiOverview/UpdateWizards/Concept/Index.rst

```diff
@@ -0,0 +1,44 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-concept:
+
+=============================
+The concept of update wizards
+=============================
+
+Update wizards are single PHP Classes which provide an automated way to update certain
+parts of a TYPO3 installation. There parts might be database or file system related.
+
+Wizards should be provided to ease updates for integrators and administrators. They
+are an addition to the database migration which is handled by the Core based on
+:file:`ext_tables.sql`.
+
+The execution order is not defined. Each administrator is able to execute wizards and
+migrations in any order. They are also completely skippable.
+
+Each wizard is able to check pre-conditions to prevent execution, if nothing has to
+be updated. The wizard can log information and executed SQL statements, which can be
+displayed after execution.
+
+Different types
+===============
+
+There are two types of possible update wizards:
+
+One is a generic update wizard which is very flexible as it consists of a PHP class
+and therefore can contain any PHP code.
+
+The second one is the row updater. This update wizard can only handle TCA records
+inside the database.
+
+Best practice
+=============
+
+Each extension can provide as many update wizards as necessary. Each one should do
+exactly one specific update.
+
+Examples
+========
+
+The TYPO3 core itself provides update wizards inside
```

Comment:
Maybe good to explicitly state that any extension **in** the core **should** put its update wizards in the path typo3/sysext/install/Classes/Update.

Reply:
I'm not sure whether this docs is about what core developers should do. We better should discuss that with the core team.

Until now I can not remember any place in the docs where something tells the core developer what to do.